03_scenario.adoc

```diff
@@ -124,7 +123,7 @@ Create a `rhtelab` development folder on your laptop
 mkdir rhtelab && cd rhtelab
 ----
 
-Create a parent `pom.xml` file as the `hal` client tool is not able to create it and add the following content:
+Create a parent `pom.xml` file (since currently `hal` doesn't automatically create it) and add the following content to it:
```

Comment:
I would remove `currently` because I don't think that `hal` should ever create such a pom… :)

Reply:
We can always leave users of the lab with hope :)

Reply:
The issue has more to do with how s2i works (or rather doesn't) than the tool to interact with the cluster. What needs to be fixed is not to automatically create a parent pom, rather, it's to remove s2i altogether :)

Reply:
Without a doubt! However users don't need to know that, they just need to know that the future holds great things for them :P